lib/internal/async_hooks.js

```diff
@@ -332,6 +339,39 @@ function emitDestroyScript(asyncId) {
 }
 
 
+// This is the equivalent of the native push_async_ids() call.
+function pushAsyncIds(asyncId, triggerAsyncId) {
+  const stackLength = async_hook_fields[kStackLength];
+  if (stackLength >= async_hook_fields[kStackCapacity])
```

Comment:
Is this needed?  You can just check `async_ids_stack.length`, right?

Reply:
… right, yes. :) Done!